### DISCUSSION TITLE
Compatibility with LoraWAN gateway and LoRaWAN concentrator

### DISCUSSION THREAD

**Discussion by aluz123** [LoRaMesher/LoRaMesher#38] [General]
Wondering if there's any compatibility with loraWAN gateways that use LoRaWAN concentrators. Specifically the Draguino DLOS8 that uses a  SX1301 LoRaWAN concentrator. 1 x SX1301 + 2 x 1257 LoRa Tranceiver 

https://www.dragino.com/products/lora-lorawan-gateway/item/160-dlos8.html

The DLOS8 uses semtech packet forwarder and fully compatible with LoRaWAN protocol. It includes a SX1301 LoRaWAN concentrator, which provide 10 programmable parallel demodulation paths.

DLOS8 has pre-configured standard LoRaWAN frequency bands to use for different countries. User can also customized the frequency bands to use in their own LoRaWAN network.

DLOS8 can communicate with ABP LoRaWAN end node without LoRaWAN server. System integrator can use it to integrate with their existing IoT Service without set up own LoRaWAN server or use 3rd party LoRaWAN service.

**Comment by PureTek-Innovations** [comment]
I don't maintain this library, but I do use it.
The underlying radio library, https://github.com/jgromes/RadioLib, does not support SX1301s
And I'm pretty sure that the hardware in the DLOS8 does not support [FreeRTOS](https://freertos.org/index.html).
This library needs both of these, see the dependencies section of the readme

**Comment by aluz123** [comment]
@PureTek-Innovations I appreciate the response, thank you. Do you by chance know of any Lora mesh libraries that work with gateways? I'm looking to develop a solution that uses nodes and gateways as a mesh-based remote sensor monitoring system.

**Comment by PureTek-Innovations** [comment]
If you go to section IV of LoraMesher's ["Implementation of a LoRa Mesh Library"](https://ieeexplore.ieee.org/document/9930341) there are details of a number of other solutions.

Or you could use this library, with a gateway node that sends data to LoraWAN, probably using two radios or more easily two ESP32 lora modules communicating over serial or similar with one connected to the mesh network and the other connected to LoRaWAN

**Comment by Jaimi5** [comment]
Thanks for the responses @PureTek-Innovations!! 
At this time, integrating LoRaWAN compatibility is not within our immediate objectives. We are currently focusing our efforts on other developments and improvements within the library. 
We appreciate your understanding and will keep the community updated if our priorities shift to include this feature in the future!

Thanks for the interest in this library!
